### ISSUE TITLE
How to document object params with particular properties

### ISSUE THREAD

**Issue by askmike** [sutoiku/jsdox#88]
I think I must be missing something here but I cannot figure out how to properly document object parameters (with specific properties). This is some documented code using jsdoc3:

    /**
     * Transition the globe from its current position
     * to the new coordinates.
     *
     * @param  {Object} pos - the position
     * @param  {Float} pos.lat - latitude position
     * @param  {Float} pos.lon - longtitute position
     * @return  {this}
     */
    api.center = function(pos) {
      target = calculate2dPosition(pos);
      return this;
    }

Will output:

> ### Globe.center(pos, pos.lat, pos.lon) 
>
> Transition the globe from its current position to the new coordinates.
>
> **Parameters**
> **pos**: `Object`, the position
> **pos.lat**: `Float`, latitude position
> **pos.lon**: `Float`, longtitute position
> **Returns**: `this`

The first line is what I am not really happy about (this function only has a single parameter which is an object). After looking at the code it seems like something that is easily fixed by filtering the params [before constructing the `paramsString`](https://github.com/sutoiku/jsdox/blob/28ef5546e5f2535fbb67f6ec79df31d6f13680ef/lib/analyze.js#L104), but I am pretty sure that I am missing something obvious here.

**Comment by psq** [comment]
@askmike I don't think you are missing anything.  This part of the spec wasn't supported yet, and your patch looks good, although I'm not quite sure why the tests failed on the ci build.  Will look into it.
